### DISCUSSION TITLE
Azure Multi-Tenant Setup

### DISCUSSION THREAD

**Discussion by josh48202** [OpenIDC/mod_auth_openidc#632] [Q&A]
Currently I have mod_auth_openidc working with a single tenant and it works perfectly however when i try to get it to work in a multi-tenant scenario using the OIDCValidateIssuer Off Setting and removing the OIDCProviderMetadataURL setting i get the following error...

> oidc_authorization_request_set_cookie: the number of existing, valid state cookies (7) has exceeded the limit (7), no additional authorization request + state cookie can be generated, aborting the request

The only mention to the OIDCValidateIssuer setting is in the auth_openidc.conf file added from #475.

I also want to note i have setup my app registration to be multi-tenant as well.

Has anyone else had luck setting this up with Multi-Tenant? Am i missing something other than the OIDCValidateIssuer Setting?

**Comment by zandbelt** [comment]
it looks like your requests are looping, the logs can tell you what is happening exactly
